### PR TITLE
Give the correct link of docker swarm init

### PR DESCRIPTION
**Pull request by lixiaobing1**
Give the correct link of docker swarm init

### CODE REVIEW

File: docs/swarm/swarm-mode.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ and try the [swarm mode tutorial](swarm-tutorial/index.md).
 
 When you run the command to create a swarm, the Docker Engine starts running in swarm mode.
 
-Run [`docker swarm init`](/engine/reference/commandline/swarm_init.md)]
+Run [`docker swarm init`](../reference/commandline/swarm_init.md)]
```
Comment (thaJeztah) [inline docs/swarm/swarm-mode.md]: oh, there's a trailing `]` here; can you remove that as well?

**Comment by lixiaobing10051267** [comment]
@thaJeztah ok, the reduntant "]" has been removed.

**Comment by SvenDowideit** [comment]
LGTM @sfsmithcha

**Comment by sfsmithcha** [comment]
LGTM

**Pull request merged**